homeassistant/components/fivem/strings.json

```diff
@@ -11,7 +11,8 @@
     },
     "error": {
       "cannot_connect": "Failed to connect. Please check the host and port and try again. Also ensure that you are running the latest FiveM server.",
-      "invalid_gamename": "The api of the game you are trying to connect to is not a FiveM game."
+      "invalid_game_name": "The api of the game you are trying to connect to is not a FiveM game.",
+      "unknown_error": "[%key:common::config_flow::error::unknown%]"
     },
     "abort": {
       "already_configured": "FiveM server is already configured"
```

Comment:
Maybe use this reference string as value?
https://github.com/home-assistant/core/blob/b6ad79e2b893683e6788336e850563f8325fcfc1/homeassistant/strings.json#L67

Reply:
Both for strings.json and en.json or just strings.json?

Reply:
We don't need to change the translation strings at all as they are updated by our infrastructure daily from Lokalise. The `strings.json` is the template that we need to change. But we allow updating en.json in PRs to make it easier for developers to test and see changes by using the frontend when working on the PR.

There's a script to sync the English translation with `strings.json` locally:
```
python -m script.translations develop
```